### PR TITLE
Reducers, actions, and components tests 

### PR DESCRIPTION
**Pull request by artemstepanenko**

### CODE REVIEW

File: src/components/BugsDiff/BugsDiff.test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import TestUtils from 'react-dom/test-utils';
+import { PureBugsDiff } from './BugsDiff';
+
+const { describe, it, expect } = global;
```
Comment (gkatsanos) [inline src/components/BugsDiff/BugsDiff.test.js]: anyway to set this in 1 file once and for all instead of repeating it?

Comment (artemstepanenko) [inline src/components/BugsDiff/BugsDiff.test.js]: Wouldn't I need to import that file everywhere?

File: src/components/BugsHistory/BugsHistory.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+describe('BugsHistory component', () => {
+  const widget = TestUtils.renderIntoDocument(
+    <RenderableBugsHistory period="Last 0 Days" />,
+  );
+  const header = TestUtils.findRenderedDOMComponentWithClass(
+    widget,
+    'panel-heading',
+  );
+  it('displys "Last 0 Days" in the header', () => {
```
Comment (gkatsanos) [inline src/components/BugsHistory/BugsHistory.test.js]: typo

File: src/components/BugsHistory/BugsHistory.test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import TestUtils from 'react-dom/test-utils';
+import { PureBugsHistory } from './BugsHistory';
+
+const { describe, it, expect } = global;
```
Comment (gkatsanos) [inline src/components/BugsHistory/BugsHistory.test.js]: you can remove that one when setting the ESlint globals ;)

Comment (artemstepanenko) [inline src/components/BugsHistory/BugsHistory.test.js]: Fixed.

File: src/components/BugsHistory/BugsHistory.test.js (hunk 2 above)
Comment (artemstepanenko) [inline src/components/BugsHistory/BugsHistory.test.js]: Fixed.

**Pull request merged**